model/src/main/kotlin/config/FileStorageConfiguration.kt

```diff
@@ -48,7 +48,11 @@ data class FileStorageConfiguration(
         }
 
         return httpFileStorage?.let { httpFileStorageConfiguration ->
-            HttpFileStorage(httpFileStorageConfiguration.url, httpFileStorageConfiguration.headers)
+            HttpFileStorage(
```

Comment:
Commit message:

> base Url and the query string. Furthermore, mask the query string for printing

"base URL"

> Furthermore, mask the query string for printing only if it is not empty. Otherwise it is not printed at all. Apply this to headers in the HttpFileStorageConfiguration as well.

I'd suggest instead: "Also, mask the query string only if it is not empty to be able to omit empty query strings from the serialization."